corehq/apps/reports/filters/users.py

```diff
@@ -142,10 +142,12 @@ def user_type_tuple(self, t):
         )
 
     def location_tuple(self, location):
+        location_id = location.location_id
         text = location.get_path_display()
-        if self.display_types:
+        if self.namespace_locations:
+            location_id = f'l__{location_id}'
             text = f'{text} [location]'
-        return ("l__%s" % location.location_id, text)
+        return (location_id, text)
```

Comment:
Are you sure about this one?  I'm guessing all the old references which were just updated weren't using the EMWF class parsers to interpret the request.